programs/server/config.xml

```diff
@@ -523,6 +522,31 @@
     </query_masking_rules>
     -->
 
+    <http_handlers>
+        <root_handler/>
+        <ping_handler>/ping</ping_handler>
```

Comment:
That way of configuring doesn't look intuitive.

A more common approach is something like a 'router' (see apache / nginx / jsexpress / react), and for each particular URL you define the handler. Currently, the order of applying handlers is not clear, and no clear way to resolve the ambiguity. Usually, it is checked from top to bottom, so that configuration suggests that root_handler should be applied always unconditionally. 

I think  something like that would be more  intuitive:
```xml
<http_handlers>
     <!-- rules are checked from top to bottom, first match runs the handler -->
     <rule>
           <url>/</url> <!-- match requests on that url -->
           <method>POST,GET</method> <!-- and with that method -->  
           <!-- other possible request matching rules can be introduced, for headers, ips, etc -- >
           <query_param>query</query_param> <!-- todo: how to match several query params? how to match param with value? -->
           <handler>
               <type>dynamic_query_handler</type>
               <query_param_name>query</query_param_name>
          </handler>
     </rule>
     <rule>
           <url>/ping</url>
           <handler><type>ping_handler</type></handler>
     </rule>
     <rule>
           <url>/</url>
           <handler><type>root_handler</type></handler>
     </rule>
     <rule>
           <url>/</url>
           <handler><type>root_handler</type></handler>
     </rule>

     <rule>
           <url>/static_page</url>
           <handler>
              <type>staticfile_handler</type>
               <content-type>text/html</content-type>
              <filename>index.html</filename><!-- absolute or relative / locked inside user files folder?  --> 
          </handler>
     </rule>


     <rule>
           <url>/my_query_results</url>
           <method>GET</method>
           <handler>
              <type>predefined_query_handler</type>
              <queries>
                 ....
              </queries>
          </handler>
     </rule>

     <rule> <!-- no conditions, i.e. match  everything -->
           <handler>
              <type>static</type>
              <status>404</status>
              <content-type>text/html</content-type>
              <body><h1>Check your query</h1></body>
          </handler>
     </rule>
</http_handlers>
```